packages/rum/src/boot/startRecording.ts

```diff
@@ -23,14 +25,21 @@ export function startRecording(
     httpRequest ||
     createHttpRequest(configuration, configuration.sessionReplayEndpointBuilder, SEGMENT_BYTES_LIMIT, reportError)
 
-  const { addRecord, stop: stopSegmentCollection } = startSegmentCollection(
-    lifeCycle,
-    configuration,
-    sessionManager,
-    viewContexts,
-    replayRequest,
-    encoder
-  )
+  let addRecord = (_record: BrowserRecord) => {}
+  let stopSegmentCollection = noop
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏to avoid this noop, we could setup a cleanupTasks array like in startRum